jujugui/static/gui/src/app/components/inspector/test-inspector.js

```diff
@@ -44,4 +49,25 @@ describe('Inspector', function() {
           <div>child</div>
         </div>);
   });
+
+  it('calls the changeState callable when on header click', function() {
```

Comment:
I don't think that this is the appropriate test here because you are actually testing that header component not the inspector.  "passes changeState callable to header component"